docs/NativeModulesAndroid.md

```diff
@@ -109,11 +109,13 @@ class AnExampleReactPackage implements ReactPackage {
 The package needs to be provided to the ReactInstanceManager when it is built. See `UIExplorerActivity.java` for an example. The default package when you initialize a new project is `MainReactPackage.java`.
 
 ```java
+// <your react-native app>/MainActivity.java
 mReactInstanceManager = ReactInstanceManager.builder()
   .setApplication(getApplication())
   .setBundleAssetName("AnExampleApp.android.bundle")
   .setJSMainModuleName("Examples/AnExampleApp/AnExampleApp.android")
   .addPackage(new AnExampleReactPackage())
+  .addPackage(new ToastPackage())           <-- provide toast package
```

Comment:
This can be helpful. Somewhere i saw ``compositePackage``.

Not sure what is the best way to register packages: 
1) chain addPackages function calls
2) use compositePackage(Not sure)